.travis.yml

```diff
@@ -9,12 +9,14 @@ addons:
   postgresql: "9.6"
 before_script:
   - sleep 10
+  - psql -c 'create database tolaactivity;' -U postgres
 install:
   - cat requirements/base.txt | grep "^Django==\|^psycopg2" | xargs pip install
   - pip install -r requirements/pkg.txt
 script:
   - python manage.py test --tag=pkg
   - pip install -r requirements/ci.txt
+  - TOLA_DB_NAME=tolaactivity python manage.py makemigrations --check --dry-run
```

Comment:
At docker-compose-dev.yml file database name is tola_activity. 
`    environment:
      POSTGRES_DB: tola_activity
      POSTGRES_USER: root
      POSTGRES_PASSWORD: root`

Reply:
Yes, but Travis is not using `docker-compose-dev.yml`, it's using it's own way to build. This is why we need to put it here.